pallets/ajuna-board/src/guessing.rs

```diff
@@ -43,6 +43,10 @@ where
 		}
 	}
 
+	fn get_last_player(_state: &Self::State) -> Self::Player {
+		unimplemented!()
```

Comment:
Isn't it possible to implement this?  Not sure I like the `unimplemented` here

Reply:
Yep I'll implement and rebase this branch. Thanks